pkgs/by-name/so/sobjectizer/tests/ping-pong-shared/default.nix

```diff
@@ -0,0 +1,20 @@
+{
+  cmake,
+  lib,
+  stdenv,
+  sobjectizer
+}:
+
+stdenv.mkDerivation rec {
+  name = "ping-pong-shared";
+  src = ./.;
+  nativeBuildInputs = [
+    cmake
+    sobjectizer
+  ];
+
+  doCheck = true;
+  checkPhase = ''
+    ./${name} -r 1000
+  '';
```

Comment:
```suggestion
  checkPhase = ''
    runHook preInstall

    ./ping-pong-shared -r 1000
    
    runHook postInstall
  '';
```

Reply:
@Sigmanificient shouldn’t it be `runHook preCheck` and `runHook postCheck`?

Reply:
Yes!